src/arviz_plots/plots/pair_plot.py

```diff
@@ -0,0 +1,452 @@
+"""Pair plot code."""
+from collections.abc import Mapping, Sequence
+from copy import copy
+from importlib import import_module
+from typing import Any, Literal
+
+import numpy as np
+import xarray as xr
+from arviz_base import rcParams, xarray_sel_iter
+from arviz_base.labels import BaseLabeller
+
+from arviz_plots.plot_matrix import PlotMatrix
+from arviz_plots.plots.dist_plot import plot_dist
+from arviz_plots.plots.utils import (
+    filter_aes,
+    get_group,
+    process_group_variables_coords,
+    set_grid_layout,
+)
+from arviz_plots.visuals import (
+    label_plot,
+    labelled_x,
+    labelled_y,
+    remove_axis,
+    remove_matrix_axis,
+    scatter_couple,
+)
+
+
+def plot_pair(
+    dt,
+    var_names=None,
+    filter_vars=None,
+    group="posterior",
+    coords=None,
+    sample_dims=None,
+    marginal=True,
+    marginal_kind="kde",
```

Comment:
default to rcparam or even none and pass to plot_dist so plot_dist changes none to the rcparam